components/Cards/Cards.css

```diff
@@ -0,0 +1,174 @@
+@import url("../Avatar/Avatar.css");
+
+.cards-container{
+    position: absolute;
+    margin-left: 5.875rem;
+    margin-top: 6.25rem;
+    margin-right: auto;
+}
+.main-menu{
+    height: 470.75rem;
+}
+.button-margin-top{
+    margin-top: 6.25rem;
+    margin-bottom: 3.75rem;
+}
+.card-container-horizontal {
+    width: 29.375rem;
+    height: 25rem;
+    position: relative;
+    left: 1.438rem;
+    border-bottom: 1px solid var(--card-bottom-radius);
+}
+.cards-with-shadow-container{
+    width: 29.375rem;
+    height: 25rem;
+   top: 9.375rem;
+    position: relative;
+    left: 0rem;
+    border-bottom: 1px solid var(--card-bottom-radius);
+}
+.Text-only-cards-container{
+    top: 9.375rem;
+}
+.card-component-horizontal{
+    width: 100%;
+}
+.card-component-vertical-heading{
+    color: var(--card-primary-color);
+    top: 0.938rem;
+    right: 4.375rem;
+    font-size: 1.875rem;
+}
+.card-component-content {
+    position: absolute;
+    background-color: var(--white);
+    bottom: 2%;
+    padding: 0.625rem;
+    right: 3%;
+    font-size: 1.563rem;
+    padding: 5px;
+    color:  var(--card-bottom-radius);
+    font-size: x-large;
+}
+.fa-bg {
+    background-color: var(--white);
+    margin-left: 1.875rem;
+}
+.card-component-vertical-description {
+    right: 5px; 
+    bottom: 13;
+}
+.vertical-card-button-watch{
+    background-color: var(--primary-color);
+    position: absolute;
+    bottom: 0px;
+    left: -9.375rem;
+    font-size: 1.188rem;
+    color: var(--white);
+    font-weight: bold;
+    size: 1.25rem;
+    padding: 0.5rem 0.5rem;
+    cursor: pointer;
+}
+#card-button-watch {
```

Comment:
Grabbing an element with its id is not a good practice. Try doing this with classes in place of IDs.

Reply:
Thanks for the suggestion . I have updated that .